tox.ini

```diff
@@ -105,7 +105,7 @@ commands_pre =
     poetry install --only main,charm-libs,integration
 
     # Set the testing host before starting the lxd cloud
-    sudo sysctl -w vm.max_map_count=262144 vm.swappiness=0 net.ipv4.tcp_retries2=5
+    sudo sysctl -w vm.max_map_count=262144 vm.swappiness=0 fs.file-max=1048576
```

Comment:
Any reason why `net.ipv4.tcp_retries2` is removed?

Reply:
Likewise, we can apply that sysctl in any of the cases (lxc, vm)